### DISCUSSION TITLE
[SIP] Proposal for ARM Support

### DISCUSSION THREAD

**Discussion by CaseyLeask** [apache/superset#24071] [Ideas]
#### Motivation

I've moved our Kubernetes cluster onto [t4g.large](https://aws.amazon.com/ec2/instance-types/t4/) as part of cost and performance improvements we wished to have.

Since we are running Superset in that Kubernetes cluster, I began porting the application over and had very specific issues with [psycopg2-binary==2.9.5](https://github.com/psycopg/psycopg2/issues/1360), which have since been fixed in https://github.com/apache/superset/commit/dab038c00551bb6cd3d01889435216e902d0d513

Other than that, we are building docker images and running Apache Superset on Kubernetes in ARM instances without issue.

#### Proposed Change

Since I'm unfamiliar with the build tooling referenced in https://github.com/apache/superset/blob/master/CONTRIBUTING.md#test-environments, I'm proposing that the support or integration testing add an `arm64` CPU Architecture to the list of instances that Superset is tested with. 

#### New or Changed Public Interfaces

The addition of [multi-platform images](https://docs.docker.com/build/building/multi-platform/) to https://hub.docker.com/r/apache/superset/tags

#### New dependencies

No new dependencies are proposed.

#### Migration Plan and Compatibility

No new migrations, but an extension of Compatibility support to include ARM CPU support.

#### Rejected Alternatives

We considered keeping Intel Kubernetes Nodes, specifically for Superset, but we overcame the technical challenges we encountered in building and running Superset on ARM.

**Comment by viniolivieri** [comment]
Up! 🚀 
Multiplatform docker images are necessary!

**Comment by zfanswer** [comment]
+1+1
